packages/cli/package.json

```diff
@@ -1,15 +1,14 @@
 {
   "name": "@batiq/cli",
   "version": "0.1.0",
-  "type": "module",
+  "type": "commonjs",
   "bin": {
     "batiq": "cli.js"
   },
   "scripts": {
-    "dev": "NODE_ENV=development ts-node src/index.ts"
+    "dev": "NODE_ENV=development ts-node -r tsconfig-paths/register src/index.ts"
   },
   "dependencies": {
-    "@batiq/generator": "*",
```

Comment:
The package is being used in cli. how does it resolves?

https://github.com/feedloop/batiq/blob/328eb8fb5fb018f17bc5c668358f955084b6ed4d/packages/cli/src/index.ts#L6

https://github.com/feedloop/batiq/blob/328eb8fb5fb018f17bc5c668358f955084b6ed4d/packages/cli/src/index.ts#L7

Reply:
missing packages are automatically added after the build by nx, it's configured here https://github.com/feedloop/batiq/blob/action-expression-codegen/packages/cli/project.json#L16

Reply:
What is the implication of adding them to `package.json`?

Reply:
IMHO, better use explicit definition

Reply:
it takes an effort to synchronize dependencies once the number of packages gets larger. Also, this changes our mindset of monorepo, instead of multiple independent packages linked together, we can think of it as a single codebase with multiple entry points. We can just import any code from another project and nx will link them and add them to dependency during the build.